src/devices/Common/System/Device/Analog/ValueChangedEventArgs.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using UnitsNet;
+
+namespace System.Device.Analog
+{
+    /// <summary>
+    /// Arguments passed in when an event is triggered by the GPIO.
+    /// </summary>
+    public class ValueChangedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ValueChangedEventArgs"/> class.
+        /// </summary>
+        /// <param name="rawValue">The raw analog sensor reading</param>
+        /// <param name="newValue">The analog sensor reading, converted to voltage.</param>
+        /// <param name="pinNumber">The pin number that triggered the event.</param>
+        /// <param name="triggerReason">The reason for the event</param>
+        public ValueChangedEventArgs(uint rawValue, ElectricPotential newValue, int pinNumber, TriggerReason triggerReason)
```

Comment:
I'd suggest changing the name of `newValue` to `value` or `electricPotential` and make sure property matches this name